docs/config.md

```diff
@@ -236,6 +236,9 @@ cpu_only_composing_models: <comma-delimited-string-list>
 # Skips the generation of detailed reports and tables
 [ skip_detailed_reports: <bool> | default: false]
 
+# Informs MA that we are profiling a LLM
+[ llm_mode: <bool> | default: false]
```

Comment:
Shouldn't all of this be going into a side branch? Otherwise there will be a new arg in main that does nothing?

Reply:
Yes. I was not merging to the correct base.